src/export.ts

```diff
@@ -13,11 +13,6 @@ export async function generateVCard(c: Contact) {
   const given_name = names[0];
   const middle_names = names.slice(1, names.length - 1).join(" ");
 
-  vcf += `N:${family_name};${given_name};${middle_names};;\n`;
-
-  if (c.website) {
-    vcf += `URL:${c.website}\n`;
-  }
```

Comment:
if you're making this only run for `"person"` cards, you probably want to move the `FN:${c.name}` and the parsing out of the names as well.

that said, since `FN` is required, for a repeater you might want to find something to put here instead (callsign? location info?)